change/@microsoft-fast-foundation-d034d172-d3c1-4cea-aea5-142253858f29.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix(select): missing aria-expanded on the template",
```

Thanks for leaning in with the new changes and adding the change file!

```suggestion
  "comment": "add missing aria-expanded to the select template",
  ```

---

Done!